server/src/main/java/org/candlepin/resource/EnvironmentResource.java

```diff
@@ -362,6 +368,12 @@ public JobDetail demoteContent(
         return contentIds;
     }
 
+    @Transactional
+    private void clearContentAccessCerts(Environment env) {
+        ownerEnvContentAccessCurator.removeAllForEnvironment(env.getId());
+        ownerEnvContentAccessCurator.refresh();
```

Comment:
How does this work?  I don't see a specific `refresh()` method in OwnerEnvContentAccessCurator, so I'm supposing it goes to `refresh(E... entities)` in the superclass.  But since you're not passing in any entities, that looks like it would just be a no-op.

Reply:
I believe it clears the cache for the curator. If it is not called, the next query through the curator returns the old versions and the content changes don't get expressed.